src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -42,9 +44,6 @@
     <ProjectReference Include="..\..\System.ObjectModel\src\redist\System.ObjectModel.depproj">
       <TargetGroup>netcore50</TargetGroup>
     </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime\src\redist\System.Runtime.depproj">
```

Comment:
There is a netcore50aot build, we should either remove that or change it to uap101 because otherwise the changes to this project may fail if they get picked up by .NET Native. We should probably make the System.ObjectModel consistent here as well to go with a straight csproj reference like System.Runtime. 

Actually looking at the package this is one of the cases where we end up using the netstandard1.3 assets for netcore50 so this change will break that configuration. In which case we either need to bump this to netstandard1.7 or explicitly add a netcore50 build.